FirebaseAppCheck/Interop/FIRAppCheckInterop.h

```diff
@@ -31,6 +31,10 @@ NS_SWIFT_NAME(AppCheckInterop) @protocol FIRAppCheckInterop
                     completion:(FIRAppCheckTokenHandlerInterop)handler
     NS_SWIFT_NAME(getToken(forcingRefresh:completion:));
 
+/// Retrieve a new limited-use Firebase App Check token
+- (void)getLimitedUseTokenWithCompletion:(FIRAppCheckTokenHandlerInterop)handler
+    NS_SWIFT_NAME(getLimitedUseToken(completion:));
+
```

Comment:
We haven't talked about this before but I think it's worth discussing: should this be marked as `optional` so if an older version of the SDK pulls this newer dependency in it doesn't fail? I'm not 100% convinced, but wanted to call it out (if someone had `FirebaseAppCheck` locked to a certain version, the interop library in CocoaPods could potentially be pulled in at a higher version?)

Reply:
I believe it must be optional. Otherwise it will be a breaking change.

Reply:
So to summarize:
- For backwards compatibility, this interop API needs to be optional 
- For forwards compatibility, the Functions podspec needs to set the lower bound of it's dependency on AppCheckInterop to be the next releasing version

Sound correct, @ryanwilson, @paulb777?

Reply:
Marked as `@optional` and also updated Function podspec to pin to AppCheckInterop `10.10`.

Reply:
Resolving to merge.